docs/mutations.md

```diff
@@ -30,6 +30,8 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 | `<=` | `<` |
 | `==` | `!=` |
 | `!=` | `==` |
+| `and` | `or` |
+| `or` | `and` |
```

Comment:
Actually shouldn't these be in the Logical Operators category?

Reply:
```
| `and` | `or` |
| `or` | `and` |
```
Should be in Logical Operators

```
| `is` | `is not` |
| `is not` | `is` |
```

Should be in Equality Operators (if we add them in this PR)

Reply:
Done